config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'users/index'
```

Comment:
[OPTIONAL]
You should consider that it is a common convention for your `users#index` or any index path to be like `users` as opposed to `users/index`.